src/safeds/ml/nn/_input_conversion.py

```diff
@@ -0,0 +1,29 @@
+from abc import ABC, abstractmethod
+from typing import Generic, TypeVar
+
+from torch.utils.data import DataLoader
```

Comment:
Move into a `TYPE_CHECKING` block.

Reply:
All of these or just the `DataLoader`?

Reply:
Just the `DataLoader`. It's fine to load globally

1. our stuff (`safeds...`),
2. anything built into Python.

If you add the `from __future__ import annotations`, `ruff` should also suggest moving this import into a `TYPE_CHECKING` block.